staging/vhost-device-gpu/src/virtio_gpu.rs

```diff
@@ -317,6 +317,7 @@ pub struct RutabagaVirtioGpu {
     pub(crate) resources: BTreeMap<u32, VirtioGpuResource>,
     pub(crate) fence_state: Arc<Mutex<FenceState>>,
     pub(crate) scanouts: [Option<VirtioGpuScanout>; VIRTIO_GPU_MAX_SCANOUTS],
+    pub(crate) shared_uuid: BTreeMap<u32, VhostUserSharedMsg>,
```

Comment:
Maybe name this `shared_uuids`? This sounds like there is only 1 of them.

Reply:
Or actually `shared_resources` should be even better - we aren't sharing uuids, we are sharing resources (by assigning them uuids) and this would make it more clear that the key of the map is the resource id.

Reply:
`shared_resources` sounds better.